### DISCUSSION TITLE
How to Use Astro and Mantine

### DISCUSSION THREAD

**Discussion by smyja** [mantinedev/mantine#4462] [Q&A]
How can I use Mantine with [Astro](https://astro.build/)

**Comment by rtivital** [comment]
I do not think that it is possible, neither with 6.x nor with 7.x alpha. I do not even think that you should use Mantine or other UI library with astro as it defeats the purpose of the framework.

**Reply by smyja** [reply]
> I do not think that it is possible, neither with 6.x nor with 7.x alpha. I do not even think that you should use Mantine or other UI library with astro as it defeats the purpose of the framework.

Wow, okay. Thanks.
